tensorflow/python/kernel_tests/transpose_op_test.py

```diff
@@ -525,6 +525,22 @@ def testError(self):
     self._testError(
         np.arange(0., 30).reshape([2, 3, 5]), [0, 1, 1], "2 is missing")
 
+  @test_util.run_v1_only("b/120545219")
```

Comment:
I hope you don't need this decorator. Can you remove it and see if the tests pass?

Reply:
Thanks @rohan100jain, Updated.